src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: record_log_event (Python agent API)
+type: apiDoc
+shortDescription: Records a log event for use in logging in context
+tags:
+  - Agents
+  - Python agent
+  - Python agent API
+metaDescription: 'Python API: This call records a log event for use in logging in context.'
+redirects:
+  - /docs/agents/python-agent/python-agent-api/recordlogevent-python-agent-api
+  - /docs/agents/python-agent/python-agent-api/record_log_event
```

Comment:
@hmstepanek usually we see redirects for deleted files, but I'm not seeing a deleted record_log_event.mdx file in this PR. Was it deleted a while back? It's not out in the wild right now.

Reply:
This is a new api function so there wouldn't be any pre-existing to redirect from. These can be removed.